tests/middleware_tests/test_parameters_name_decoding_handler.py

```diff
@@ -34,14 +34,26 @@ async def test_decodes_query_parameter_names_only():
     """
     Test that only query parameter names are decoded
     """
-    encoded_url = "https://graph.microsoft.com?%24count=true&query=%24top&created%2din=2022-10-05&q=1%2b2&q2=M%26A&subject%2ename=%7eWelcome&%24empty"
-    expected_url = "https://graph.microsoft.com?$count=true&query=%24top&created-in=2022-10-05&q=1%2b2&q2=M%26A&subject.name=%7eWelcome&$empty"
-
+    encoded_decoded = [
+    ("http://localhost?%24select=diplayName&api%2Dversion=2", "http://localhost?$select=diplayName&api-version=2"),
```

Comment:
can we keep a case with a valueless query parameter? ($empty in the previous version)